<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminWindow</class>
 <widget class="QMainWindow" name="AdminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="currrenTimeLabel">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>540</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Current Time:</string>
    </property>
   </widget>
   <widget class="QLabel" name="actualTimeLabel">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>540</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Time</string>
    </property>
   </widget>
   <widget class="QLabel" name="adminLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>501</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Administration Panel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>521</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="editUser">
       <property name="text">
        <string>Edit User</string>
       </property>
       <property name="icon">
        <iconset resource="img.qrc">
         <normaloff>:/img/img/username.png</normaloff>:/img/img/username.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="createUserButton">
       <property name="text">
        <string>Create User</string>
       </property>
       <property name="icon">
        <iconset resource="img.qrc">
         <normaloff>:/img/img/new.png</normaloff>:/img/img/new.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetUserPasswordButton">
       <property name="text">
        <string>Reset User Password</string>
       </property>
       <property name="icon">
        <iconset resource="img.qrc">
         <normaloff>:/img/img/passkey.png</normaloff>:/img/img/passkey.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="DeleteUser">
       <property name="text">
        <string>Delete User</string>
       </property>
       <property name="icon">
        <iconset resource="img.qrc">
         <normaloff>:/img/img/wrong.png</normaloff>:/img/img/wrong.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableView" name="usernameView">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>43</y>
      <width>121</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="everythingElse">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>43</y>
      <width>401</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Logout">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>125</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Logout</string>
    </property>
    <property name="icon">
     <iconset resource="img.qrc">
      <normaloff>:/img/img/logout.png</normaloff>:/img/img/logout.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="admin">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>250</y>
      <width>221</width>
      <height>237</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="img.qrc">:/img/img/admin.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>800</width>
      <height>237</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="img.qrc">:/img/img/background.png</pixmap>
    </property>
   </widget>
   <zorder>background</zorder>
   <zorder>currrenTimeLabel</zorder>
   <zorder>actualTimeLabel</zorder>
   <zorder>adminLabel</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>usernameView</zorder>
   <zorder>everythingElse</zorder>
   <zorder>Logout</zorder>
   <zorder>admin</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>542</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuLanguage">
    <property name="title">
     <string>Language</string>
    </property>
    <addaction name="actionSpanish"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
   <addaction name="menuLanguage"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSpanish">
   <property name="text">
    <string>Spanish</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
